app/grandchallenge/components/tasks.py

```diff
@@ -979,3 +981,33 @@ def civ_value_to_file(*, civ_pk):
         )
         civ.value = None
         civ.save()
+
+
+@shared_task(**settings.CELERY_TASK_DECORATOR_KWARGS["acks-late-2xlarge"])
+def validate_voxel_values(*, civ_pk):
+    civ = get_model_instance(
+        pk=civ_pk,
+        app_label="components",
+        model_name="componentinterfacevalue",
+    )
+
+    if civ.image.segments is None:
+        image_file = civ.image.files.first()
```

Comment:
What about image that are `mhd` and `zraw` pairs? You need both.

TIFF and DZI need to be excluded.

Reply:
Aren't these guaranteed to be mha and not mhd/zraw though?

Reply:
No, both exist.